library/LogLevel.re

```diff
@@ -0,0 +1,19 @@
+type t =
+  | Quiet
+  | Error
+  | All;
+
+let toString = logLevel =>
+  switch (logLevel) {
+  | Quiet => "quiet"
+  | Error => "error"
+  | All => "all"
+  };
+
+let fromString = logLevelString =>
+  switch (logLevelString) {
+  | "quiet" => Quiet
+  | "error" => Error
+  | "all" => All
+  | _ => All
```

Comment:
I'm torn between `_ => All` to `_ => failwith("Unsupported level: " ++ logLevelString)`. What do you think? should we fail?

Reply:
p.s. just came back from vacation 😜

Reply:
I agree. I initially operated according to another tool that didn't fail on invalid level, but now that I think of it again we should fail.
I'll make the change, although it will take me a couple of days to get back to the code and modify+add test

Reply:
That's cool 😄
Thanks 😸

Reply:
As I understand the code, I think this code path is unreachable, since CmdLiner already does the validation according to this: https://github.com/ohana54/fnm/blob/497d8e1351cd0a8ad4e0c47baa95db2a934c8716/executable/FnmApp.re#L205-L207

I will still change it to fail, in case this function will be used in a different code path in the future, but I can't test it with an e2e test (I can write a unit for this function only if that's preferred)

Reply:
You’re right. Tests isn’t necessary in this case. Sorry!

Reply:
It's all good :)
Pushed the fix.